sdk/batch/batch-typespec/routes.tsp

```diff
@@ -67,6 +93,8 @@ API.
 }
 
 #suppress "@azure-tools/typespec-azure-core/casing-style" "The names of Model types must use camelCase"
+#suppress "@azure-tools/typespec-azure-core/no-rpc-path-params" "Operations defined using RpcOperation should not have path parameters."
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Operation 'Exists' should be defined using a signature from the Azure.Core namespace."
```

Comment:
Can we suppress this kind of error globally?

Reply:
Unfortunately not - Don't think in our experience TypeSpec typically offers a way to globally surpress something. Either hopefully once the TypeSpec team has a solution to the ResourceExists template, we won't need to surpress this anymore.